community-tutorials/pterodactyl-panel-install-ubuntu/01-en.md

```diff
@@ -0,0 +1,313 @@
+
+```bash
+Is this user an administrator? (yes/no) [no]:
+ > yes
+
+ Email Address:
+ > example@example.com
+
+ Username:
+ > example
+
+ First Name:
+ > Example
+
+ Last Name:
+ > User
+
+Passwords must be at least 8 characters in length and contain at least one capital letter and number.
+If you would like to create an account with a random password emailed to the user, re-run this command (CTRL+C) and pass the `--no-password` flag.
+
+ Password:
+ > ********
+
++----------+--------------------------------------+
+| Field    | Value                                |
++----------+--------------------------------------+
+| UUID     | 26eaeeab-0ee5-4d62-aa46-6626cedea3fc |
+| Email    | example@example.com                  |
+| Username | example                              |
+| Name     | Example User                         |
+| Admin    | Yes                                  |
++----------+--------------------------------------+
+```
+
+# Step 2.2 - (SSL Configuration)
+To secure your Panel it is necessary to configure ssl. For this purpose a certbot container is included with the setup.
+
+To generate your ssl certificate files run:
+```
+docker-compose run --rm --service-ports certbot certonly -d <domain>
+```
+
+This will request a certificate from letsencrypt and stores it inside ``conf.d/letsencrypt/``. After requesting your certificate you need to add it to your panels configuration. Modify the following lines:
+```bash
+nano conf.d/panel.env
+```
+
+```diff
+- SSL_CERT="/etc/certs/cert.pem"
+- SSL_CERT_KEY="/etc/certs/cert.key"
++ SSL_CERT="/etc/letsencrypt/live/<domain>/fullchain.pem"
++ SSL_CERT_KEY="/etc/letsencrypt/live/<domain>/privkey.pem"
+
+APP_URL=http://<domain>/ 
+```
+
+Lets encrypt will automatically manage your certificates and regulary request new ones when they're about to expire.
+
+Now recreate your panel's container to apply the changes:
+
+```bash
+docker-compose down && docker-compose up -d
+```
+
+
+You can access your Panel now by opening ``https://<domain>/``. Login using your credentials configured earlier.
+
+
+# Step 3 - (Daemon configuration)
+To configure your daemon you need to open the administrator tab on the web-ui
+
+![admin tab](admin-tab.jpg)
+
+To configure your new node you need to add a new location first. To do so open the **Locations** tab and click **Create New**
+
+![create new location](create-new-location.jpg)
+
+Fill in the information and click Create:
+
+![Create new location create](create-new-location-data.jpg)
+
+
+To configure your new node navigate to the **Nodes** tab and click **Create New**.
+
+![create new node](create-new-node.jpg)
+
+Here you need to fill in all the required information. Tailor the values to your need. It's important to configure the domain the same as used for your letsencrypt configuration!
+
+![configure new node](configure-new-node.jpg)
```

Comment:
I had to fill all fields.

Reply:
Modified the statement to "Here you need to fill in all fields".